src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,442 @@ authors: [oluoluoxenfree, alextait1]
+
+##### Screen reader only text
+
+A common technique that developers often employ to supply additional information for screen reader users is to use CSS to visually hide a passage of text such that it will be announced by a screen reader, but not visually present in the interface. Since display:none and visibility:hidden both prevent content from being present in the accessibility tree, there is a common “hack” involving a chunk of CSS code that will accomplish this. The most common CSS class names for this code snippet (both by convention and throughout libraries like bootstrap) are `‘sr-only’` and `‘visually-hidden’`. We found that 13.31% of desktop pages and 12.37% of mobile pages had one or both of these CSS class names.
+
+##### Announcing Dynamically Rendered Content
+
+One of the biggest accessibility challenges in modern web development is handling dynamically rendered content which is everywhere in interfaces. The presence of new or updated things in the DOM often needs to be communicated to screen readers. Some thought needs to be put into which updates need to be conveyed. For example, form validation errors need to be conveyed where as a lazy loaded image may not. There also needs to be done in a way that is not disruptive to a task in progress.
+
+One tool we have to help with this is aria-live regions. Live regions allow us to listen for changes in the DOM, such that the updated content can be announced by a screen reader. Typically the `aria-live` attribute is placed on its own container element that is already present in the DOM rather than an element that is dynamically rendered. It is important to determine a dedicated node in the DOM that has no chance of being dynamically manipulated by other factors for the live region, ensuring that the announcements are reliable. When elements within this container dynamically render or update (for example, status updates or notification that a form was not successfully submitted) the changes will be announced. 
+
+We found that 16.84% of desktop pages and 15.67% of mobile pages have live regions. This attribute has three potential values; “polite”, “assertive” and “off”. Typically the “polite” value is used, partly because it is the default value, but also because the announcement of the dynamic content will only happen once the user stops interacting with the page which in many cases is the desired user experience rather than interrupting their input. If a status update is critical enough, use “assertive” and it  will disrupt the screen reader’s current speech queue. If it is set to “off” the announcement will not happen. It is important that the natural screen reader experience and flow be respected and that the “assertive” announcements be reserved for extreme cases, and not used for things like marketing announcements.  
```

Comment:
```suggestion
We found that 16.84% of desktop pages and 15.67% of mobile pages have live regions. This attribute has three potential values: `polite`, `assertive`, and `off`. Typically the `polite` value is used, partly because it is the default value, but also because the announcement of the dynamic content will only happen once the user stops interacting with the page. In many cases this is the desired user experience, rather than interrupting their input. If a status update is critical enough, use `assertive` and it  will disrupt the screen reader’s current speech queue. If it is set to `off` the announcement will not happen. It is important that the natural screen reader experience and flow be respected and that the `assertive` announcements be reserved for extreme cases, and not used for things like marketing announcements.  
```